docs/run-node/interact-node.md

```diff
@@ -0,0 +1,49 @@
+<!--
+order: 3
+-->
+
+# Interacting with the Node
+
+## Pre-requisite Readings
+
+- [Running a Node](./run-node.md) {prereq}
+
+## Via CLI
+
+Now that your chain is running, it is time to try sending tokens from the first account you created to a second account. In a new terminal window, start by running the following query command:
+
+```bash
+simd query account $(simd keys show my_validator -a) --chain-id my-test-chain
+```
+
+You should see the current balance of the account you created, equal to the original balance of `stake` you granted it minus the amount you delegated via the `gentx`. Now, create a second account:
+
+```bash
+simd keys add receiver
+```
+
+The command above creates a local key-pair that is not yet registered on the chain. An account is registered the first time it receives tokens from another account. Now, run the following command to send tokens to the second account:
```

Comment:
```suggestion
The command above creates a local key-pair that is not yet registered on the chain. An account is created the first time it receives tokens from another account. Now, run the following command to send tokens to the `receiver` account:
```

Reply:
I'd call it the `recipient`. Not a big deal either way

Reply:
FTR `recipient` belongs to a finance-friendly terminology, so...